omero/sysadmins/limitations.txt

```diff
@@ -109,14 +118,16 @@ group that data is in into a private group.
 File format support
 -------------------
 
-DeltaVision OMX files (.hdr) not viewable
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Large images with floating-point pixel data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Pyramids of image tiles are currently not generated for images with
+floating-point pixel data, meaning the imported image will be scrambled if it
+is over a certain size. This primarily affects the following file formats:
```

Comment:
Defaults are set by https://www.openmicroscopy.org/site/support/omero5.1/sysadmins/config.html#omero-pixeldata-max-plane-height and https://www.openmicroscopy.org/site/support/omero5.1/sysadmins/config.html#omero-pixeldata-max-plane-width